assets/javascripts/discourse/widgets/docked-post.js.es6

```diff
@@ -8,7 +8,10 @@ class QuickPostCooked extends PostCooked {
     const $html = $(`<div class='cooked'>${this.attrs.cooked}</div>`);
     this._insertQuoteControls($html);
     this._showLinkCounts($html);
-    this._fixImageSizes($html);
+
+    // removed inhttps://github.com/discourse/discourse/commit/e305365168528883872d4ce9efd109e41149ef0a
```

Comment:
This was broken in `main` for this repo, I could not do this fix without commenting this out, a further fix will need to be made here (or it could possibly be removed altogether, I didn't look into it)